Makefile

```diff
@@ -199,6 +199,10 @@ build/src/code/fmodf.o: OPTFLAGS := -g
 build/src/code/__osMemset.o: OPTFLAGS := -g
 build/src/code/__osMemmove.o: OPTFLAGS := -g
 
+build/src/code/code_800EC960.o: CFLAGS += -signed
+build/src/code/code_800F7260.o: CFLAGS += -use_readwrite_const
+build/src/code/code_800F9280.o: CFLAGS += -use_readwrite_const
```

Comment:
Can you add comments on what these do, and why they're needed?
(basically what you wrote for the PR description)

Reply:
Done (assuming the wording is what you had in mind)